src/main.rs

```diff
@@ -107,7 +112,7 @@ struct ExitPanic(i32);
 /// as our final act.
 struct ExecPanic(std::process::Command);
 
-fn main() -> Result<(), VetError> {
+fn main() -> Result<(), ()> {
```

Comment:
I don't understand why we're bypassing the normal termination handling here.

That being said, if we want to handle every result and `std::process::exit` manually, we can have `main()` return nothing, and handle the error ourself I suppose.

It feels to me like we should just return a `miette::Report` here.